Jenkinsfile

```diff
@@ -175,6 +175,7 @@ pipeline {
                                 sh """
                                     set +x
                                     . \$MDSPLUS_DIR/setup.sh
+                                    export PYTHONPATH=$MDSPLUS_DIR/python/
```

Comment:
This needs to be \$MDSPLUS_DIR to escape the Jenkinsfile

Reply:
Just noticed your comment.   Am assuming you mean there should be a backslash in front of the `$MDSPLUS_DIR`.   Correct?

Reply:
Based on how the Jenkins file references other environment variables, I changed the line by adding braces.   I hope that is the correct syntax change for the line.

```
export PYTHONPATH=${MDSPLUS_DIR}/python/
```

Reply:
Just noticed that the initial comment was edited to clarify the change.   Means that my second guess was wrong.   So will change the line to use the first guess, `\$MDSPLUS_DIR`.